examples/summarize_dataset.py

```diff
@@ -0,0 +1,110 @@
+import torch
```

Comment:
Eventually we'll want to put this dataset onto huggingface

Reply:
(Possibly) Relevant: https://huggingface.co/datasets/openai/summarize_from_feedback

Reply:
Actually, I used this dataset before they were public to hf, but that was for the RLHF blog post not for this PR.